test_log_statement.py

```diff
@@ -0,0 +1,19 @@
+import unittest2
+import main
+
+class TestLogStatement(unittest2.TestCase):
```

Comment:
I like splitting up the test cases so that you can see which ones are failing instead of them all being under one test case. Would that trade off outweigh readability? @g-harel

Reply:
Also if one fails at the top you won't see the ones at the bottom until the one at the top is fixed.

Reply:
Those are great points, there might be ways for you to generate test cases instead of doing all the checks in a loop inside a single test. Not super familiar with Python, but I've set this up before in Go and JavaScript. No worries if this doesn't end up panning out.